flow-typed/npm/lodash.groupby_vx.x.x.js

```diff
@@ -0,0 +1,33 @@
+// flow-typed signature: 9d825067548ae9710390f7e55f6181c5
+// flow-typed version: <<STUB>>/lodash.groupby_v^4.6.0/flow_v0.68.0
+
+/**
+ * This is an autogenerated libdef stub for:
+ *
+ *   'lodash.groupby'
+ *
+ * Fill this stub out by replacing all the `any` types.
+ *
+ * Once filled out, we encourage you to share your work with the
+ * community by sending a pull request to:
+ * https://github.com/flowtype/flow-typed
+ */
+
+declare module 'lodash.groupby' {
+  declare module.exports: any;
+}
+
+/**
+ * We include stubs for each file inside this npm package in case you need to
+ * require those files directly. Feel free to delete any files that aren't
+ * needed.
+ */
+
+
+// Filename aliases
+declare module 'lodash.groupby/index' {
+  declare module.exports: $Exports<'lodash.groupby'>;
+}
+declare module 'lodash.groupby/index.js' {
+  declare module.exports: $Exports<'lodash.groupby'>;
+}
```

Comment:
It looks like flow-typed has definitions.
Are they not compatible with the modularised version?

https://github.com/flowtype/flow-typed/tree/master/definitions/npm/lodash_v4.x.x

Reply:
Doesn't seem to be